src/core.js

```diff
@@ -312,12 +312,13 @@
 
     // if the isn't going to play automatically and the first image is
     // loaded make sure to render it
-    if( this.$element.find("img")[0] == element &&
-        (event && event.type !== "error") &&
+    if( this.$element.find("img")[0] == element && 
         (!this.options.autoplay || !this.options.autoplay.enabled) ){
-      this.goto(0);
-      this.$element.trigger("tau.init");
-      initTriggered = true;
+      if( !event || event.type !== "error") {
```

Comment:
Two things:

1. This can still be part of the larger conjunction in the first guard, that is two nested guards without intervening code can always be conjoined.
2. As far as I can tell the important difference here is that it allows the `event` parameter to be falsey whenever `imageLoaded` is called? That seems important [here](https://github.com/johneke-akamai/tau/blob/5ac65ebdcfccd2c17a970e145f33c9891e607931/src/core.js#L298).

Reply:
Yes @johnbender, everywhere else where `imageLoaded` is called, an event is passed in. Except the one spot you mentioned. So that one spot trips up the code.

My patch effectively make sure that things still work in that one case where `imageLoaded` is called without the last `event` parameter.

This was ofcourse the easy fix :) if I missed something let me know and I can clean up the fix

Reply:
Nope! This is super great.

I'll merge and then move the condition back up into the parent conjunction.

Thanks man!